spec/schedule_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 require "rails_helper"
 require "whenever/test"
+require "open3"
```

Comment:
I never knew about this. It's awesome! 👍

Reply:
👍